### ISSUE TITLE
Capitalization of types is not allowed by xs-check - but AoE2:DE allows it

### ISSUE THREAD

**Issue by KSneijders** [Divy1211/xs-check#7]
The following XS Code is allowed by AoE2:DE (I know this shouldn't be legal, but it is...) but errors out in xs-check

# Script ([MRE](https://stackoverflow.com/help/minimal-reproducible-example))
```c
void test() {
    INT i = 3;
    Vector vec = vector(1.0,1.1,1.2);
    vector vec2 = Vector(1.0,1.1,1.2);
    xsChatData("my int" + i);
}
```

# Result of XS-CHECK:
```
Error: ParsingError
   ╭─[default0.xs:5:9]
   │
 5 │     INT i = 3;
   │         ┬
   │         ╰── found i expected '--', '++', '*', '/', '%', '+', '-', '<', '<=', '>', '>=', '==', or '!='
───╯
Finished analysing 'default0.xs'.
```

# Result in game:
✅ Allowed

Prints to chat: `my int3`

**Comment by Divy1211** [comment]
yES

**Comment by Divy1211** [comment]
I should be able to fix this, thanks for the report!

**Comment by Divy1211** [comment]
Fixed in v0.1.4
